gax-grpc/src/test/java/com/google/api/gax/grpc/GrpcOperationCallableImplTest.java

```diff
@@ -653,8 +654,8 @@ private void assertFutureFailMetaFail(
 
     assertThat(exception).isNotNull();
     if (statusCode != null) {
-      assertThat(exception.getCause()).isInstanceOf(GrpcApiException.class);
-      GrpcApiException cause = (GrpcApiException) exception.getCause();
+      assertThat(exception.getCause() instanceof ApiException);
```

Comment:
The actual assertion has to happen after the call to `assertThat`. If you just pass data to `assertThat` and do nothing with the response, then nothing will be checked.

Also, why not be more specific about the type of `ApiException` being expected?